pvnet_app/test_forecast_compiler.py

```diff
@@ -0,0 +1,70 @@
+import pytest
+import numpy as np
+import logging
+
+from forecast_compiler import validate_forecast
```

Comment:
This shoudl be `from pvnet_app.forecast_compiler import validate_forecast` I think